src/audio_worklet.js

```diff
@@ -97,6 +97,20 @@ function createWasmAudioWorkletProcessor(audioParams) {
         // Reserve space for the output data
         dataPtr += bytesPerChannel * i.length;
       }
+      if (!this.outputViews) {
+        this.outputViews = [];
+        k = outputDataPtr;
+        for (/*which output*/ i of outputList) {
+          for (/*which channel*/ j of i) {
+            this.outputViews.push({
+              // dataPtr is the sanity check (to be implemented)
```

Comment:
If its a sanity check then perhaps put it behind `if ASSERTIONS`?

Reply:
I’ve some rearranging to do, to make sure the addresses are fixed, and if those change from the base I’ll add assertions.

I’ll make this a draft and work on it next week, adding some robustness.

Reply:
> If its a sanity check then perhaps put it behind `if ASSERTIONS`?

Interestingly (unless I'm doing something hideous) but `assert()` isn't available in the `AudioWorkletGlobalScope`, and I can't add it as a dependency the same as `stackAlloc()`, etc. This stuff I'm very rusty with, mind.